website/docs/style/illustration/illustrations-list.js

```diff
@@ -32,6 +32,10 @@ const illustartionsList = {
       name: 'UnderConstruction',
       group: 'States',
     },
+    {
+      name: 'MechanismInside',
```

Comment:
What do you think if we call it something like Settings, or Broken, or maybe with the name of the state it helps to visualise? Why I'm thinking about it: all other illustrations in this group have names of the states so it's clear for the users when they can use them.
<img width="729" alt="image" src="https://github.com/user-attachments/assets/ede48f64-5b34-49c2-8ce5-d1c146ef9e89">

Reply:
sounds reasonable, I think we can call it "Configuration" (the team that wants to use it now uses it as "please configure something"